src/RequestHandle.cpp

```diff
@@ -480,4 +562,83 @@ auto curl_write_data(
     return data_length;
 }
 
+auto curl_xfer_info_old(
```

Comment:
this isn't that important but maybe just name them after the actual curl functions like all the rest of the curl_ functions? e.g. curl_xfer_info and curl_progress.  Unless if you know which one is the old/new its a bit ambiguous

Reply:
Realized today that we don't need these callbacks to abort the CURL request, so I've removed them. More in main comment.